CHANGES.txt

```diff
@@ -1,36 +1,19 @@
-0.9.0 (2016-04-15)
-------------------
+Unreleased
+----------
 
 Deprecations
 ~~~~~~~~~~~~
 
-- Python 3.2 is no longer supported by Waitress.
+- Python 2.6 is no longer supported.
 
-- Python 2.6 will no longer be supported by Waitress in future releases.
-
-Security/Protections
-~~~~~~~~~~~~~~~~~~~~
-
-- Building on the changes made in pull request 117, add in checking for line
-  feed/carriage return HTTP Response Splitting in the status line, as well as
-  the key of a header. See https://github.com/Pylons/waitress/pull/124 and
-  https://github.com/Pylons/waitress/issues/122.
-
-- Waitress will no longer accept headers or status lines with
-  newline/carriage returns in them, thereby disallowing HTTP Response
-  Splitting. See https://github.com/Pylons/waitress/issues/117 for
-  more information, as well as
-  https://www.owasp.org/index.php/HTTP_Response_Splitting.
-
-Bugfixes
+Features
 ~~~~~~~~
 
-- FileBasedBuffer and more important ReadOnlyFileBasedBuffer no longer report
-  False when tested with bool(), instead always returning True, and becoming
-  more iterator like.
-  See: https://github.com/Pylons/waitress/pull/82 and
-  https://github.com/Pylons/waitress/issues/76
+- Waitress is now able to listen on multiple sockets, including IPv4 and IPv6.
+  Instead of passing in a host/port combination you now provide waitress with a
+  space delineated list, and it will create as many sockets as required.
```

Comment:
It's probably a good idea to ensure folks that host=/port= will continue to work for X more releases (where you can decide what X is, although I suggest quite a few).